### DISCUSSION TITLE
CVE-2023-52251 | HIGH

### DISCUSSION THREAD

**Discussion by senthil13** [provectus/kafka-ui#4419] [General]
Hi Experts, 
The latest version has one vulnerability; is there any fix available?

[CVE-2023-52251](https://nvd.nist.gov/vuln/detail/CVE-2023-52251)

[link](https://nvd.nist.gov/vuln/search/results?form_type=Basic&results_type=overview&query=provectus+kafka-ui+0.7.1&search_type=all&isCpeNameSearch=false)

**Comment by Haarolean** [comment]
Hi!
Please take a look at these:
https://github.com/provectus/kafka-ui/discussions/4255
and
https://github.com/kafbat/kafka-ui/discussions/23
